### ISSUE TITLE
A group of models fail accuracy test in inference with freezing is enabled

### ISSUE THREAD

**Issue by shunting314** [pytorch/pytorch#120545]
There are a batch of models that fail the accuracy test in inference when freezing is enabled.
- TB: mnasnet1_0 , moco, shufflenet_v2_x1_0
- TIMM: adv_inception_v3, botnet26t_256, gluon_inception_v3, selecsls42b, swsl_resnext101_32x16d

They pass the test if we disable freezing.

I checked adv_inception_v3 specifically. The test
```
time python benchmarks/dynamo/timm_models.py --accuracy --inference --bfloat16 --backend inductor --disable-cudagraphs --only adv_inception_v3
```
fail if we add --freezing option.

I tried a few other things for this model
1. raise the tolerance to 4x, still fail
2. raise the tolerance to 100x, pass
3. use --float32 rather than -bfloat16, pass.

We probably need check if there is some real issue about freezing before we further raise tolerance to make the tests pass.

cc @ezyang @msaroufim @bdhirsh @anijain2305 @zou3519

**Comment by eellison** [comment]
These are all due to Conv-Batchnorm folding, which can change numerics. Commenting out the below makes them all pass. I think we should just bump the tolerance. 

```
diff --git a/torch/_inductor/fx_passes/freezing_patterns.py b/torch/_inductor/fx_passes/freezing_patterns.py
index 6c1f52f4702..2a12aca0d77 100644
--- a/torch/_inductor/fx_passes/freezing_patterns.py
+++ b/torch/_inductor/fx_passes/freezing_patterns.py
@@ -50,7 +50,7 @@ def freezing_passes(gm: torch.fx.GraphModule, aot_example_inputs):
-        binary_folding_pass.apply(gm.graph)  # type: ignore[arg-type]
+        # binary_folding_pass.apply(gm.graph)  # type: ignore[arg-type]
         ```

**Comment by shunting314** [comment]
For adv_inception_v3, I tried the following tolerance:
```
        # tolerance = 1e-1 # pass
        # tolerance = 8 * 1e-2 # pass
        # tolerance = 6 * 1e-2 # fail
        # tolerance = 4 * 1e-2 # fail
        # tolerance = 1e-2 # fail
        # tolerance = 1e-3 # fail (default)
```
I'll set tolerance as '8 * 1e-2' for those models for now. Also I'll test this with other models and call out if any model fail even with such high tolerance.

Besides raising the tolerance, I have one more question. The conv-bm fusion fuse conv with 4 possible ops (add, mul, sub, div). If I disable the fusion with (mul, div), the test with the default 1e-3 tolerance will pass. I'm wondering how useful the fusion with mul,div is?  It looks like we just move the mul/div operation from convolution's epilogue to convolution's prologue. cc @eellison
